src/index.tsx

```diff
@@ -21,85 +22,21 @@ export const preview = (parameter: any) => {
     channel.emit("preview", parameter);
 }
 
-export function previewTemplate(strings: TemplateStringsArray, ...values: any[]) {
+export function raw(name: string) {
     return new Function(`
-        var p = arguments[0];
-        var strings = ${JSON.stringify(strings)};
-        var values = ${JSON.stringify(values)};
-
-        function space(indent) {
-            var text = [];
-            for (var i = 0; i < indent; ++i) {
-                text.push(" ");
-            }
-            return text.join("");
-        }
-        function toObject(info, names) {
-            var quote = info.quote;
-            var indent = info.indent;
-            var endIndent = info.endIndent;
-            var separator = info.separator;
-            var joinSeparator = info.joinSeparator;
-            var endSeparator = info.endSeparator;
-            var objs = ["{"];
-            var length = names.length;
-
-            names.forEach(function (n, i) {
-                var v = p[n];
-
-                v = JSON.stringify(v);
-
-                objs.push(space(indent) + n + ": " + v  + (i + 1 === length ? endSeparator : separator));
-            });
-            objs.push(space(endIndent) + "}");
-            return objs.join(joinSeparator);
-        }
-        var dirty = strings.reduce(function (prev, next, i) {
-            var name = values[i];
-
-            if (typeof name === "undefined") {
-                name = "";
-            }
-            var value = name;
+        var knobs = arguments[0];
 
-            if (Array.isArray(name)) {
-                var info = name[0];
-                var names = name.slice(1);
-
-                if (info.type === "object") {
-                    value = toObject(info, names);
-                }
-            } else if (name in p) {
-                value = p[name];
-            }
-
-            if (typeof value === "object") {
-                value = JSON.stringify(value);
-            }
-            return "" + prev + next  + value;
-        }, "");
-        return dirty;
+        return knobs[${JSON.stringify(name)}];
```

Comment:
I think that `JSON.stringify` is unnecessary. `name` is string.

Reply:
@mixed 

The reason is to put quotes.